### DISCUSSION TITLE
SA Bot Dialy Transalation Update

### DISCUSSION THREAD

**Discussion by Shluffy** [ScratchAddons/ScratchAddons#2934] [Help]
I see 2 (#2916, #2924) merged on 7/06* and 2 (#2925, #2926) closed. we already have 7 for 7/07*, only 1 (#2933) of which has been merged, and the other 6 (#2927,#2928,#2929,#2930,#2931,#2932) closed. What is happening with the bot right now?
Why did 2 get merged on 7/06* if it is daily, and only 1 should be?

*All dates are in 2021

**Comment by cobaltt7** [comment]
:flushed: I made some mistakes
________________________________
From: ShluffyMonster ***@***.***>
Sent: Wednesday, July 7, 2021 6:59:16 AM
To: ScratchAddons/ScratchAddons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ScratchAddons/ScratchAddons] SA Bot Dialy Transalation Update (#2934)


I see 2 (#2916<https://github.com/ScratchAddons/ScratchAddons/pull/2916>, #2924<https://github.com/ScratchAddons/ScratchAddons/pull/2924>) merged on 7/06* and 2 (#2925<https://github.com/ScratchAddons/ScratchAddons/pull/2925>, #2926<https://github.com/ScratchAddons/ScratchAddons/pull/2926>) closed. we already have 7 for 7/07*, only 1 (#2933<https://github.com/ScratchAddons/ScratchAddons/pull/2933>) of which has been merged, and the other 6 (#2927<https://github.com/ScratchAddons/ScratchAddons/pull/2927>,#2928<https://github.com/ScratchAddons/ScratchAddons/pull/2928>,#2929<https://github.com/ScratchAddons/ScratchAddons/pull/2929>,#2930<https://github.com/ScratchAddons/ScratchAddons/pull/2930>,#2931<https://github.com/ScratchAddons/ScratchAddons/pull/2931>,#2932<https://github.com/ScratchAddons/ScratchAddons/pull/2932>) closed. What is happening with the bot right now?
Why did 2 get merged on 7/06* if it is daily, and only 1 should be?

*All dates are in 2021

**Reply by Shluffy** [reply]
well, that explains why 2 got merged yesterday, but now why sab is posting multiple

**Reply by cobaltt7** [reply]
they ran it manually
